understanding/20/audio-description-or-media-alternative-prerecorded.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Audio Description or Media Alternative (Prerecorded)</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Prerecorded videos can be understood by more people</dd>
-		<dt>Author task</dt><dd>Provide a description of the visual content in videos</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Prerecorded videos can be understood by more people.</dd>
+         <dt>What to do</dt><dd>Provide a description of the visual content in videos.</dd>
+         <dt>Why it's important</dt><dd>People who are blind and some with low vision can't see the visual content.</dd> 
```

Comment:
@rainbreaw commented
> Not a concern, but a recommendation for consideration, in understanding/20/audio-description-or-media-alternative-prerecorded.html, understanding/20/audio-description-prerecorded.html and understanding/20/extended-audio-description-prerecorded.html:

Consider extending why it's important to not just include people who are blind or with low vision, but also
```suggestion
         <dt>Why it's important</dt><dd>People who are blind and some with low vision can't see the visual content, and individuals with a cognitive disability that makes it difficult to understand implied meaning such as facial expressions.</dd> 
```

Reply:
I feel like this is overly specific for the in brief section. I also feel like the intent is clearly directed at "people who are blind or visually impaired." The Benefits section does include "people who have difficulty perceiving or understanding moving images". 

I'm going to suggest as a possible modification:
```suggestion
         <dt>Why it's important</dt><dd>People who are blind or who cannot understand the visual content.</dd> 
```
It may be a good idea to consider creating a PR to add in more specifics in the Benefits section.
That said, it opens up a broader discussion on the priorities of audio description, and gets us to something I've talked about before, and that is the notion that a one-size-fits-all audio description is much less desirable than something that could be personalized for someone's needs. Most who are blind infer a lot from tone of voice and context and would probably find it disruptive if audio descriptions painstakingly described the emotional state of people speaking on screen (except where plot advancement depends on them understanding critical reaction shots that are only conveyed visually).

Reply:
@GreggVan commented on my in-process and incomplete second attempt on rewording this (not preserved). I've since modified, but I wanted to capture his suggestion here

> like your proposed new version very much.   Excellent.    I think it needs to be a sentence though.  And it might be a bit clearer that people may not understand what is visually happening even if they have vision.   What do you think about
```suggestion
         <dt>Why it's important</dt><dd>To make videos understandable by people who are blind or who cannot understand the visual content for other reasons including cognitive.</dd> 
```

Reply:
Doing a slight modification, since "to them" infers audio descriptions, but a script or whatever is also allowable
```suggestion
         <dt>Why it's important</dt><dd>People who are blind or who cannot understand the visual content can have it described.</dd> 
```